docs/user_guide.md

```diff
@@ -72,7 +72,17 @@ Administrator can add "SysAdmin" role to an ordinary user by toggling the switch
 ![browse project](img/set_admin_remove_user.png)
 
 ##Pulling and pushing images using Docker client
+
 **NOTE: Harbor only supports Registry V2 API. You need to use Docker client 1.6.0 or higher.**  
+
+If you encounter an error as below when you pull or push images, you need to add '--insecure-registry' option to /etc/default/docker (ubuntu) or /etc/sysconfig/docker (centos):    

```

Comment:
Please add more information to explain to user that currently by default nginx server registry API with HTTP protocol not HTTPS